modules/@apostrophecms/asset/lib/webpack/utils.js

```diff
@@ -276,7 +371,57 @@ async function verifyBundlesEntryPoints (bundles) {
   return packedFilesByBundle;
 };
 
-function formatConfigs (chain, webpackConfigs) {
+// Normalize config:
```

Comment:
OK thanks for this comment.

Reply:
Comment on reBundle could just refer to this function to explain its options.

Reply:
This is an old function, merging and transforming webpack configs.